src/commonMain/kotlin/app/cash/turbine/Turbine.kt

```diff
@@ -88,18 +91,26 @@ public operator fun <T> Turbine<T>.plusAssign(value: T) { add(value) }
 /**
  * Construct a standalone [Turbine].
  */
-public fun <T> Turbine(): Turbine<T> = TurbineImpl()
+public fun <T> Turbine(timeoutMs: Long? = null): Turbine<T> = TurbineImpl(timeoutMs = timeoutMs)
```

Comment:
`timeout: Duration` please! You should be able to keep safe types all the way down now, I think. There definitely was a `delay` overload but I'm not sure about `withTimeout`.